community/blog/2022-02 Property based testing the Arcus Python implementations for streamlined ML development/README.md

```diff
@@ -0,0 +1,150 @@
+# Property-based testing the Arcus Python implementations for streamlined ML development
+Arcus.ML contains several easy-to-use Python implementations for more fluent ML development. This post will guide through the process of testing these implementations with a technique to ensure robust outcomes: property-based testing.
```

Comment:
```suggestion
Arcus.ML contains several easy-to-use Python implementations for more fluent machine learning development. This post will guide through the process of testing these implementations with a technique to ensure robust outcomes: property-based testing.
```

Maybe better to not use an abbreviation?

Reply:
👍 Yes, is good!